section3/exercises/arrays.rb

```diff
@@ -18,50 +18,76 @@
 # Store the array in a variable.
 animals = ["Zebra", "Giraffe", "Elephant"]
 print animals
+puts ""
 
 # EXAMPLE: Write code below that will print "Zebra" from the animals array
 # YOUR CODE HERE
-print animals[0]
+puts '-' * 10
+puts animals[0]
 
 # YOU DO: Write code below that will print the number of elements in array of
 # animals from above.
-
+puts ""
+puts animals.count
 
 # YOU DO: Write code that will reassign the last item in the animals
 # array to "Gorilla"
-
-
+puts ""
+animals[2] = "Gorilla"
+puts "position 2 has been changed to #{animals[2]}"
+puts '-' * 10
 # YOU DO: Write code that will add a new animal (type of your choice) to position 3.
-
+animals.push("Fish")
+print animals
+puts ""
+puts '-' * 10
 
 # YOU DO: Write code that will print the String "Elephant" in the animals array
-
-
+puts animals[2]
+puts "the string 'elephant' was replaced by 'Gorilla' as instructed in line 33-37'..."
+puts '-' * 10
 #-------------------
 # PART 2: Foods: Array Methods
 #-------------------
 
 # YOU DO: Declare a variable that will store an an array of at least 4 foods (strings)
 
+foods = ["pizza", "pasta", "burgers", "pudding"]
 
 # YOU DO: Write code below that will print the number of elements in the array of
 # foods from above.
 
+puts "There are #{foods.count} elements in the array 'foods'"
 
 # YOU DO: Write code below that uses a method to add "broccoli" to the foods array and
 # print the changed array to verify "broccoli" has been added
 
+foods.push("broccoli")
+puts "new array: #{foods}"
 
 # YOU DO: Write code below that removes the last item of food from the foods array and
 # print the changed array to verify that item has been removed
+puts ""
+foods.pop
+puts "new array: #{foods}"
+puts '-' * 10
 
-
-# YOU DO: Write code to add 3 new foods to the array. 
+# YOU DO: Write code to add 3 new foods to the array.
   # There are several ways to do this - choose whichever you'd like!
 # Then, print the changed array to verify the new items have been added
-
+foods.unshift("ice cream")
+foods.push("oranges")
+foods.push("hot dogs")
```

Comment:
I like `push` personally, but you'll also see the `Array#<<` method _a lot_ (same operation, different 'alias')